lib/SimpleSAML/IdP.php

```diff
@@ -505,8 +505,10 @@ public function handleLogoutRequest(array &$state, $assocId)
 
         $this->authSource->logout($returnTo);
 
-        $handler = $this->getLogoutHandler();
-        $handler->startLogout($state, $assocId);
+        if ($assocId !== null) {
```

Comment:
Twelve lines above we also test for `$assocId` being `null`, so I'm guessing this could be a valid situation